// _@vuepress_core@1.2.0@@vuepress/core/.temp/internal/routes.js
/**
 * Generated by "@vuepress/internal-routes"
 */

import { injectComponentOption, ensureAsyncComponentsLoaded } from '@app/util'
import rootMixins from '@internal/root-mixins'
import GlobalLayout from "/Users/shaohao/tianbao/tianbaoliu.github.io/node_modules/_@vuepress_core@1.2.0@@vuepress/core/lib/client/components/GlobalLayout.vue"

injectComponentOption(GlobalLayout, 'mixins', rootMixins)
export const routes = [
  {
    name: "v-484e6a70",
    path: "/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-484e6a70").then(next)
    },
  },
  {
    path: "/index.html",
    redirect: "/"
  },
  {
    name: "v-f18bb016",
    path: "/en/npm.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-f18bb016").then(next)
    },
  },
  {
    name: "v-0407a85a",
    path: "/en/ecma.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-0407a85a").then(next)
    },
  },
  {
    name: "v-7dac3257",
    path: "/en/scratch.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-7dac3257").then(next)
    },
  },
  {
    name: "v-7f72c49a",
    path: "/en/skulpt.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-7f72c49a").then(next)
    },
  },
  {
    name: "v-3177d7b3",
    path: "/scratch/sb3.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-3177d7b3").then(next)
    },
  },
  {
    name: "v-36c5231e",
    path: "/en/webpack.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-36c5231e").then(next)
    },
  },
  {
    name: "v-4e8bacd3",
    path: "/scratch/1.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-4e8bacd3").then(next)
    },
  },
  {
    name: "v-9c3e2e5a",
    path: "/scratch/links.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-9c3e2e5a").then(next)
    },
  },
  {
    name: "v-6fd5699a",
    path: "/share/env.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-6fd5699a").then(next)
    },
  },
  {
    name: "v-14ff4f7d",
    path: "/share/eslint.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-14ff4f7d").then(next)
    },
  },
  {
    name: "v-50325b83",
    path: "/share/javascript-basic.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-50325b83").then(next)
    },
  },
  {
    name: "v-b9bc2a9e",
    path: "/share/md.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-b9bc2a9e").then(next)
    },
  },
  {
    name: "v-21a75d1a",
    path: "/share/opt.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-21a75d1a").then(next)
    },
  },
  {
    name: "v-29cadf1a",
    path: "/share/webpack-opt.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-29cadf1a").then(next)
    },
  },
  {
    name: "v-6aad30d3",
    path: "/share/wechat.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-6aad30d3").then(next)
    },
  },
  {
    name: "v-c9c0209a",
    path: "/share/webpack.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-c9c0209a").then(next)
    },
  },
  {
    name: "v-22822741",
    path: "/url/fe.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-22822741").then(next)
    },
  },
  {
    name: "v-73d46815",
    path: "/skulpt/1.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-73d46815").then(next)
    },
  },
  {
    name: "v-2173f94c",
    path: "/share/js/%E5%87%BD%E6%95%B0%E4%BD%9C%E7%94%A8%E5%9F%9F%E5%92%8C%E5%9D%97%E4%BD%9C%E7%94%A8%E5%9F%9F.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-2173f94c").then(next)
    },
  },
  {
    path: "/share/js/函数作用域和块作用域.html",
    redirect: "/share/js/%E5%87%BD%E6%95%B0%E4%BD%9C%E7%94%A8%E5%9F%9F%E5%92%8C%E5%9D%97%E4%BD%9C%E7%94%A8%E5%9F%9F.html"
  },
  {
    name: "v-15e64e85",
    path: "/skulpt/links.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-15e64e85").then(next)
    },
  },
  {
    name: "v-467e35e5",
    path: "/share/optimize.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-467e35e5").then(next)
    },
  },
  {
    name: "v-66188ef3",
    path: "/share/optimize2.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-66188ef3").then(next)
    },
  },
  {
    name: "v-4ef9519a",
    path: "/share/scaffold.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-4ef9519a").then(next)
    },
  },
  {
    name: "v-57a5fa7b",
    path: "/share/webpack-note.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-57a5fa7b").then(next)
    },
  },
  {
    path: '*',
    component: GlobalLayout
  }
]
